lucet-wasi/src/main.rs

```diff
@@ -237,6 +248,17 @@ fn run(config: Config<'_>) {
             .build()
             .expect("instance can be created");
 
+        if let Some(timeout) = config.timeout {
+            let kill_switch = inst.kill_switch();
+            thread::spawn(move || {
+                thread::sleep(timeout);
+                // We may hit this line exactly when the guest exits, so sometimes `terminate` can
+                // fail. That's still acceptable, so just ignore the result.
+                #[allow(unused_must_use)]
```

Comment:
this attribute isn't preventing this warning, yet if I move it right above `thread::spawn` it works. must be some weird interaction between attributes and the grammar

Reply:
I've since learned the "cool" trick to `.ok()` a Result to ignore it - I think the warning changing is a result of building this on 1.35 vs nightly